app/backplane/radio_module/src/c_lora_receive_tenant.cpp

```diff
@@ -68,8 +68,26 @@ void CLoraReceiveTenant::FlightRun() {
     // Should not receive anything while in flight
 }
 
+static void shutoffTimerExpirationFn(k_timer* timer) {
+    LOG_INF("Shutting off cameras");
+    CGpio gpios[4] = {
+        CGpio(GPIO_DT_SPEC_GET(DT_ALIAS(gpio0), gpios)),
+        CGpio(GPIO_DT_SPEC_GET(DT_ALIAS(gpio1), gpios)),
+        CGpio(GPIO_DT_SPEC_GET(DT_ALIAS(gpio2), gpios)),
+        CGpio(GPIO_DT_SPEC_GET(DT_ALIAS(gpio3), gpios))
+    };
+
+    for (auto & gpio : gpios) {
+        gpio.SetPin(0);
+    }
+}
+
 void CLoraReceiveTenant::LandedRun() {
     // Should not receive anything while landed
+    CSoftTimer shutoffTimer{shutoffTimerExpirationFn, nullptr};
```

Comment:
Won't this 
- A. block the state machine task for ten minutes (not that big a deal if theres nothing else on that thread but afaik kinda against the ideas of tasks and tenants *(or wait actually do we want to transmit from the ground bc this is on the lora task and so we will only get transmissions every ten minutes. it doesnt look likee transmitting on the ground is written but that seems like something useful to have so we can get landing coordinates)* and 
- B. repeat and shutoff gpios once every ten minutes bc the task will call StateMachine::Clock(), sleep ten minutes, wake up, shut off, return, then call Clock again and repeat. Like shutting things off multiple times isnt that big a deal bc its just off and then off again but this seems like not the best way to go about this

Reply:
We don't need to block until the timer expires I guess. Could just have a boolean flag that gets updated in the callback so Ill make that modification

Reply:
Improved the logic